mysite/settings.py

```diff
@@ -39,6 +39,7 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'blog',
+    'multiselectfield',
```

Comment:
Remove this line too (do we need both mysite/settings.py and blog/settings.py?)

Reply:
While working on deployment, Bill and I created a duplicate settings.py file because we thought that was contributing to the error we encountered while deploying. We never really figured it out but he wanted to keep both copies just in case, we can probably discuss this more next week